index.js

```diff
@@ -42,7 +44,8 @@ function bindSSHConnection(config, server, netConnection) {
             config.dstHost,
             config.dstPort, function(err, sshStream) {
                 if (err) {
-                    throw err;
+                    server.emit('error', err);
+                    return;
```

Comment:
@mrfelton i guess were having trouble because of the return here. 

you may check the develop branch